src/Card.js

```diff
@@ -0,0 +1,12 @@
+
+
+class Card {
+  constructor(id, question, answers, correctAnswer) {
+    this.id = id;
+    this.question = question;
+    this.answers = answers;
+    this.correctAnswer = correctAnswer;
```

Comment:
Does this have to take strings in as parameters? Using the objects in the data files seems much more straight forward.